ROMFS/px4fmu_common/init.d/rc.interface

```diff
@@ -227,6 +240,17 @@ then
 		then
 			pwm disarmed -c ${PWM_ACHDIS} -p ${PWM_AUX_DISARMED} -d ${OUTPUT_AUX_DEV}
 		fi
+		
+		#
+		# Per channel disarmed settings
+		#
+		pwm disarmed -c 1 -p p:PWM_AUX_DIS1 -d ${OUTPUT_AUX_DEV}
+		pwm disarmed -c 2 -p p:PWM_AUX_DIS2 -d ${OUTPUT_AUX_DEV}
+		pwm disarmed -c 3 -p p:PWM_AUX_DIS3 -d ${OUTPUT_AUX_DEV}
+		pwm disarmed -c 4 -p p:PWM_AUX_DIS4 -d ${OUTPUT_AUX_DEV}
+		pwm disarmed -c 5 -p p:PWM_AUX_DIS5 -d ${OUTPUT_AUX_DEV}
+		pwm disarmed -c 6 -p p:PWM_AUX_DIS6 -d ${OUTPUT_AUX_DEV}
```

Comment:
MAIN and AUX are applied inconsistently. For MAIN you only run `pwm disarmed -c X ...` if PWM_OUT is set. For AUX it's done regardless of PWM_AUX_OUT. Is this intentional?

MAIN and AUX aren't consistent here. For MAIN we set this if